loopchain/blockchain/votes/v0_1a/vote.py

```diff
@@ -69,36 +70,39 @@ def result(self) -> ExternalAddress:
 
     # noinspection PyMethodOverriding
     @classmethod
-    def new(cls, signer: Signer, timestamp: int,
-            block_height: int, old_leader: ExternalAddress, new_leader: ExternalAddress) -> 'LeaderVote':
-        return super().new(signer, timestamp,
-                           block_height=block_height, old_leader=old_leader, new_leader=new_leader)
+    def new(cls, signer: Signer, timestamp: int, block_height: int, round_: int,
+            old_leader: ExternalAddress, new_leader: ExternalAddress) -> 'LeaderVote':
+        return super().new(signer, timestamp, block_height=block_height, round_=round_,
+                           old_leader=old_leader, new_leader=new_leader)
 
     # noinspection PyMethodOverriding
     @classmethod
-    def empty(cls, rep: ExternalAddress, block_height: int, old_leader: ExternalAddress):
-        return cls(rep, 0, Signature.empty(), block_height, old_leader, ExternalAddress.empty())
+    def empty(cls, rep: ExternalAddress, block_height: int, round_: int, old_leader: ExternalAddress):
+        return cls(rep, 0, Signature.empty(), block_height, round_, old_leader, ExternalAddress.empty())
 
     @classmethod
     def _deserialize(cls, data: dict):
         data_deserialized = super()._deserialize(data)
         data_deserialized["block_height"] = int(data["blockHeight"], 16)
+        data_deserialized["round_"] = data["round_"]
```

Comment:
Need to name a key with underbar_?

Reply:
The `round` is a reserved keyword in python builtin.

Reply:
I pointed that the name `round` is possible if it is used as a key of `dict`, not a bare usage.

```python
>>> a = {"round": 1}
>>> a["round"]
1
```